packages/cli/src/lib/implementation/core-config.model.ts

```diff
@@ -7,13 +7,13 @@ export type PersistConfigCliOptions = {
   'persist.format': Format;
 };
 
+/* eslint-disable @typescript-eslint/naming-convention */
 export type UploadConfigCliOptions = {
   'upload.organization': string;
   'upload.project': string;
   'upload.apiKey': string;
   'upload.server': string;
 };
-/* eslint-enable @typescript-eslint/naming-convention */
```

Comment:
This should be reverted. ESLint directives don't work on blocks of code, only lines. An `eslint-disable` comment affects all subsequent lines until the end of the file. To disable only a specific block, but not following blocks, it needs to be wrapped in `eslint-disable` and `eslint-enable` comments.

```ts
enabled here 

/* eslint-disable @typescript-eslint/naming-convention */
disabled here
disabled here too
/* eslint-enable @typescript-eslint/naming-convention */

enabled here
```

Your change disables the rule for the whole file (and the second comment is actually redudant):

```ts
enabled here 

/* eslint-disable @typescript-eslint/naming-convention */
disabled here
/* eslint-disable @typescript-eslint/naming-convention */
still disabled 

disable here too
```

Reply:
Ho! ok thx! will revert it!